unitTests/ui/DynamicFacet/DynamicFacetRangeTest.ts

```diff
@@ -80,13 +80,20 @@ export function DynamicFacetRangeTest() {
       expect(test.cmp.facetType).toBe(FacetType.numericalRange);
     });
 
-    it(`when not setting a valueFormat option
-      should set it number`, () => {
+    it(`when not setting a valueFormat option with a field containing "date"
+      should set it to "date"`, () => {
+      options.field = '@cooldatefield';
+      initializeComponent();
+      expect(test.cmp.options.valueFormat).toBe(DynamicFacetRangeValueFormat.date);
+    });
+
+    it(`when not setting a valueFormat option with a field not containing "date"
```

Comment:
Similar comment as above for the first "not" on this line,

Reply:
valueFormat is not set but field is set, updated description